spec/services/manifest_builder_spec.rb

```diff
@@ -439,18 +439,6 @@ def logical_structure(file_set_id)
       end
     end
 
-    context "when in staging and pyramidals are disabled" do
```

Comment:
What happens now in this case?

Reply:
Nothing. There's no such thing as disabling pyramidal access anymore.